### PR TITLE
Resize & hide/show facet panel

### PR DESCRIPTION
**Pull request by samrichca**
- add size & variant to Button component
- add resize to facet panel
- add show/hide to facet panel
- increase contrast for disabled buttons [ticket here](https://github.com/overture-stack/dms-jbrowse-components/issues/18)

### CODE REVIEW

File: components/ScrollToTop.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
+/*
```
Comment (samrichca) [inline components/ScrollToTop.tsx:1]: this is from the previous PR

File: components/Button.tsx [hunk 2]
```diff
@@ -19,51 +19,113 @@
...
     align-items: center;
     cursor: pointer;
     position: relative;
     &:hover {
-      background-color: ${theme.colors.accent_dark};
+      background: ${getButtonTheme(theme)[variant].hover.background};
     }
     &:disabled,
     &:disabled:hover {
-      background-color: ${theme.colors.grey_4};
```
Comment (samrichca) [inline components/Button.tsx:48]: Patrick asked for changes to the button disabled state

File: components/pages/explorer/Facets.tsx [hunk 3]
```diff
@@ -187,6 +189,23 @@ const Facets = () => {
...
 		>
+			<div
+				css={css`
+					padding: 10px;
+				`}
+			>
+				<Button variant="secondary" size="sm" onClick={hidePanel}>
+					<Chevron
+						style={css`
+							transform: rotate(90deg);
```
Comment (justincorrigible) [inline components/pages/explorer/Facets.tsx:200]: 👍

File: public/static/chevron.svg
Comment (justincorrigible) [inline public/static/chevron.svg:1]: This file may not be necessary, since you have the tsx icon, no?

Comment (samrichca) [reply public/static/chevron.svg:1]: now that you mention it the whole folder doesn't need to be there. I'll delete it on the next PR that spins off this one.

**Review by justincorrigible — APPROVED** [review]

**Pull request merged**